test/e2e-common.sh

```diff
@@ -353,6 +352,18 @@ function install_knative_serving_standard() {
     fi
   fi
 
+  # Well, installing Knative with Ambassador is a sequential process, i.e. Knative serving needs to be installed before
+  # Ambassador. This is because Ambassador only enables Knative support when ingresses.networking.internal.knative.dev
+  # CRD is present, which is not the case if Ambassador is installed before Knative serving. This is reflected in the
+  # user facing docs, but these tests install Ambassador before Knative serving leading to test failures.
```

Comment:
> but these tests install Ambassador before Knative serving

This isn't quite true.  We install the CRDs (incl. KIngress), then we install the networking layer (all of which needs the CRDs to exist), then we install the rest of serving.  This has changed a bit over time, so perhaps this is no longer needed?

Reply:
Well, you're right! I've removed this piece. There's something else going on in the tests that I cannot get my head around! :man_facepalming: